automarker_2/automarker_app/lib/steps.py

```diff
@@ -28,6 +29,19 @@ def get_all_file_paths(directory):
             yield os.path.join(path, filename)
 
 
+def is_current_os_windows():
+    return platform.system() == "Windows"
+
```

Comment:
Let's get rid of this two liner, we can do this check inside: `get_python_executable_path` like this

```python
def get_python_executable_path():
    if platform.system() == "Windows"
        return "Scripts/python"
    return "bin/python"
```